scripts/scripts-1.js

```diff
@@ -0,0 +1,171 @@
+function getMinNumberFromParams(numb_1, numb_2, numb_3, numb_4) {
+
+  const minimalNumber = Math.min(numb_1, numb_2, numb_3, numb_4);
+
+  if (numb_1 === undefined || numb_2 === undefined || numb_3 === undefined || numb_4 === undefined) {
+
+    return 'Undefined value!';
+
+  } else if (numb_1 === '' || numb_2 === '' || numb_3 === '' || numb_4 === '') {
+
+    return 'Your value is empty';
+
+  } else if (isNaN(minimalNumber)) {
+
+    return 'Your value is not a number!';
+
+  }
+
+  else
```

Comment:
```suggestion
```
You can remove the last `else` because if there will be any validation errors in your `if`s function will stop execution and will return the value from the corresponding condition